### DISCUSSION TITLE
no signal (being desperate)

### DISCUSSION THREAD

**Discussion by thomaskilian** [john30/ebusd#1289] [Q&A] [answered]
Hi there, after having to suspend my activities for half a year I now started to connect the  ebus adapter - with no success. The command ebusctl i returns
version: ebusd 23.3.23.3-57-gd45c5a77
update check: revision 23.3 available
device: /dev/ttyACM0, serial high speed, enhanced
signal: no signal
reconnects: 0
masters: 1
messages: 11
conditional: 0
poll: 0
update: 4
address 31: master #8, ebusd
address 36: slave #8, ebusd

What is happening? The adapter is conncted via USB (I can connect via terminal manually) and the green led is flashing according to the activity on the bus. So, why "no signal"?

**Comment by Koky05** [comment]
How do you run your ebusd and what is your configuration?

**Reply by thomaskilian** [reply]
I run it on an old Raspi and as you can see the config points to the usb where the adaoter is connected.  The relevant line in the config is
EBUSD_OPTS="--scanconfig -d ens:/dev/ttyACM0 --latency=50 --lograwdata"
The sw is pulled from the recent git and compiled locally.

**Reply by thomaskilian** [reply]
No takers? When I found this adapter I thought it was a neat way to get around making my own (which for just listening would have been not so difficult). But now I seem to be forced to dump the ebud-adapter and get back to square one? Isn't there even a way to connect via that easy-interface to see telegrams?

**Comment by thomaskilian** [comment, marked as answer]
I made a step forward. The connection type "Serial" needed to be turned on via the web-interface (which was hidden under a folded option). Now I have the signal acquired. Seems I have a start now.
